src/common/playlists/PlayMeta.jsx

```diff
@@ -1,8 +1,10 @@
 import { cloneElement } from "react";
 import { Helmet } from "react-helmet";
+import * as Plays from '../../plays'
 
 function PlayMeta({ id, name, description, path, cover, component }) {
   const playFolder = path.split("/")[2];
+  const RenderPlay = Plays[component]
```

Comment:
@Angryman18 So this eventually import all the plays and then pick the component def from it.

As we have the component name already, can't we just translate it to the required component def and pass as we were doing before?

Reply:
the componet name is here coming as string. i wonder how we link the string to the exact componet we have in plays folder. we might wanna discuss this also since im not getting any idea to fix how you wanted.